### ISSUE TITLE
Proposals in cycle 6 are not allowing votes to be cast

### ISSUE THREAD

**Issue by dantudor** [navcoin/navcoin-core#498]
Using the proposal below as an example,
State is showing as `0` (pending), and status is showing as `expired`

Should this proposal allow voting or should it progressed pass state `0`?

```
{
  "version": 2,
  "hash": "86d521a55e402fe2a34ff72a96af9c43108d6726fe134eb3521d4d0210459e0e",
  "blockHash": "f49141110f93643ac7e0f4b7720ba921fa7e10434d02e5d7614534f54eb80bb7",
  "description": "Social Media Marketing + Crypto Advertising for NAV 2019 - https://collab.navcoin.org/content/perma?id=1116",
  "requestedAmount": "22500.00",
  "notPaidYet": "22500.00",
  "userPaidFee": "50.00",
  "paymentAddress": "NNXposEe2cTQKg1MsKbV8nMpSAjxk7F3Z1",
  "proposalDuration": 2592000,
  "votesYes": 0,
  "votesNo": 0,
  "votingCycle": 6,
  "status": "expired",
  "state": 0
}
```

**Comment by mxaddict** [comment]
I think this is fixed in 5.0

**Comment by mxaddict** [comment]
Just reopen if not.
